kritis-charts/templates/postinstall/1_job.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Values.postinstall.job.name }}
+  namespace: {{ .Values.serviceNamespace }}
+  annotations:
+    "helm.sh/hook": "post-install"
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded"
+  labels:
+    {{ .Values.kritisInstallLabel }}: ""
+spec:
+  template:
+    metadata:
+      name: {{ .Values.postinstall.job.name }}
+      labels:
+        app: {{ .Values.postinstall.job.name }}
+    spec:
+      restartPolicy: Never
+      containers:
+        - name: {{ .Values.postinstall.job.name }}
+          image: "{{ .Values.repository }}{{ .Values.postinstall.job.image }}:{{ .Values.image.tag }}"
```

Comment:
`.Values.image.tag` will be common for all these images. Right? So, we will have to update tags of all images in case if we are updating one image. 

Personally, would like to have different tags value in `values.yaml` but since Kritis is already using one tag for everything, we can follow current pattern.